### PR TITLE
Sweep /legal and clean up indentation

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- Sweep /legal and clean up indentation
- Remove any redundant classes

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal](http://0.0.0.0:8001/legal)
- Verify everything is as it should

### CODE REVIEW

File: templates/legal/bootstack/index.html [hunk 1]
```diff
@@ -38,4 +38,5 @@
     </div>
   </div>
 </div>
+
```
Comment (pmahnke) [inline templates/legal/bootstack/index.html]: missing /section

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html [hunk 2]
```diff
@@ -2,125 +2,72 @@
...
       <section>
         <h2 id="service">2. Service</h2>
-
         <p>Details about the Service can be found here: <a href="https://www.ubuntu.com/server/livepatch">www.ubuntu.com/server/livepatch</a></p>
-
         <p>The Service provides users of Ubuntu with a subscription to use the Software on up to three physical or virtual Ubuntu systems and to make a reasonable number of copies of this software for backup and installation purposes.</p>
       </section>
-
       <section>
         <h2 id="account">3. Account</h2>
```
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: row col-8?

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html [hunk 3]
```diff
@@ -2,125 +2,72 @@
...
         </ul>
-
-        <p>If you do not have an account you will need to create one. You are responsible for choosing an appropriate password for your accounts and for keeping such password secure. Canonical will not ask you for your password and you should not reveal it
-          to anyone. You are responsible for keeping your account details up to date.</p>
-
+        <p>If you do not have an account you will need to create one. You are responsible for choosing an appropriate password for your accounts and for keeping such password secure. Canonical will not ask you for your password and you should not reveal it to anyone. You are responsible for keeping your account details up to date.</p>
       </section>
-
       <section>
         <h2 id="end-of-service">4. End of Service</h2>
```
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: row col-8?

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html [hunk 4]
```diff
@@ -2,125 +2,72 @@
...
         <ul>
           <li class="p-list__item">we cease to make the Service (or any part thereof) available</li>
           <li class="p-list__item">by us at any time as described in <a href="#account">Section 3 above</a></li>
         </ul>
-
         <p>We may also cease to offer the services for any other reason, in which case we will provide you with notice on your account page.</p>
       </section>
-
       <section>
         <h2 id="intellectual property">5. Intellectual property</h2>
```
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: row col-8?

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html [hunk 5]
```diff
@@ -2,125 +2,72 @@
...
-
         <p>You have a non-exclusive, non-transferable (to the extent permitted by law) right to view, access and use the Services for such time as it is made available by us strictly in accordance with these Terms of Service.</p>
-
         <p>You will not acquire any rights to the Service (or the intellectual property rights contained therein) from your use of the Service, other than as set out in these Terms of Service.</p>
       </section>
-
       <section>
         <h2 id="personal-data">6. Personal data</h2>
```
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: row col-8?

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html [hunk 6]
```diff
@@ -2,125 +2,72 @@
...
+        <p>Our <a href="/legal/terms-and-policies/privacy-policy">Privacy Policy</a> explain how we treat your personal data and protect your privacy when using our services. In addition, if you use the Service for more than three physical or virtual Ubuntu systems Canonical may contact you directly, using the <span class="caps">SSO </span>account details provided, to discuss ongoing support requirements.</p>
+        <p>We may also collect certain non-personally-identifiable information, which is located on your computer. The information collected may include statistics relating to how often data is transferred, and performance metrics in relation to software and configuration. You agree this information may be retained and used by Canonical.</p>
+        <p>Canonical may disclose any or all personal data and content you have sent, posted or published if required to comply with applicable law or the order or requirement of a court, administrative agency or other governmental body. All other use of your personal data is subject to the Privacy Policy.</p>
       </section>
 
       <section>
         <h2 id="liability">7. Liability</h2>
```
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: row col-8?

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html [hunk 7]
```diff
@@ -2,125 +2,72 @@
...
-          to data. Otherwise, Canonical&#39;s total liability in contract, tort or otherwise for any claims is limited to &pound;10.</p>
-
-        <p>Nothing in these Terms of Service will exclude or limit Canonical&#39;s liability for: death or personal injury caused by the negligence of Canonical; fraud or fraudulent misrepresentation; or any other liability that cannot be excluded or limited
-          by law.</p>
+        <p>Canonical will not be liable in contract, tort or otherwise for any: indirect or consequential loss; loss of profits; loss of revenue; loss of anticipated savings; loss of business or business opportunity; loss of goodwill; or loss of or corruption to data. Otherwise, Canonical&#39;s total liability in contract, tort or otherwise for any claims is limited to &pound;10.</p>
+        <p>Nothing in these Terms of Service will exclude or limit Canonical&#39;s liability for: death or personal injury caused by the negligence of Canonical; fraud or fraudulent misrepresentation; or any other liability that cannot be excluded or limited by law.</p>
       </section>
-
       <section>
         <h2 id="general">8. General</h2>
```
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: row col-8?

File: templates/legal/terms-and-policies/privacy-policy/third-parties.html [hunk 8]
```diff
@@ -68,4 +68,5 @@
     <p class="col-8">Please submit any questions or comments about Ubuntu Online Accounts or this legal notice by contacting us at the following address: Canonical Group Ltd, 5<sup>th</sup> Floor, Blue Fin Building, 110 Southwark Street, London, England, SE1 0SU.</p>
   </div>
 </div>
+
```
Comment (pmahnke) [inline templates/legal/terms-and-policies/privacy-policy/third-parties.html]: /section?

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html [hunk 9]
```diff
@@ -2,125 +2,72 @@
...
-        account, to these Terms of Service. "you" and "your" shall refer to the company or legal entity and such users. If you do not have such authority do not accept these Terms of Service.</p>
-
+      <p>If you are entering into these Terms of Service on behalf of a company or legal entity, you represent that you have authority to bind such company or legal entity, its officers, employees and agents and all users who access the Service through the account, to these Terms of Service. "you" and "your" shall refer to the company or legal entity and such users. If you do not have such authority do not accept these Terms of Service.</p>
       <p>You must be at least 13 years old to use the Service. If you are between age 13 and 18, you confirm that you have your parent&#39;s or legal guardian&#39;s consent to use the Services and that they have read and agreed to these Terms of Service.</p>
-
       <p>Canonical may update these Terms of Service from time to time in its discretion.</p>
-
       <section>
         <h2 id="service">2. Service</h2>
```
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: row col-8?

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html (hunk 2 above)
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: row col-8?

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html [hunk 10]
```diff
@@ -2,125 +2,72 @@
...
           <li class="p-list__item">not be located in or use the Services in an <span class="caps">OFAC</span>/EAR embargoed or sanctioned country or be on the <span class="caps">U.S.</span> Commerce Department&#39;s Denied Persons List, Entity List, or Unverified List</li>
         </ul>
-
-        <p>If you do not have an account you will need to create one. You are responsible for choosing an appropriate password for your accounts and for keeping such password secure. Canonical will not ask you for your password and you should not reveal it
-          to anyone. You are responsible for keeping your account details up to date.</p>
-
+        <p>If you do not have an account you will need to create one. You are responsible for choosing an appropriate password for your accounts and for keeping such password secure. Canonical will not ask you for your password and you should not reveal it to anyone. You are responsible for keeping your account details up to date.</p>
       </section>
-
       <section>
```
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: row col-8?

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html [hunk 11]
```diff
@@ -2,125 +2,72 @@
...
-
         <ul>
           <li class="p-list__item">we cease to make the Service (or any part thereof) available</li>
           <li class="p-list__item">by us at any time as described in <a href="#account">Section 3 above</a></li>
         </ul>
-
         <p>We may also cease to offer the services for any other reason, in which case we will provide you with notice on your account page.</p>
       </section>
-
       <section>
```
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: row col-8?

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html [hunk 12]
```diff
@@ -2,125 +2,72 @@
...
         <h2 id="intellectual property">5. Intellectual property</h2>
-
         <p>You have a non-exclusive, non-transferable (to the extent permitted by law) right to view, access and use the Services for such time as it is made available by us strictly in accordance with these Terms of Service.</p>
-
         <p>You will not acquire any rights to the Service (or the intellectual property rights contained therein) from your use of the Service, other than as set out in these Terms of Service.</p>
       </section>
-
       <section>
```
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: row col-8?

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html [hunk 13]
```diff
@@ -2,125 +2,72 @@
...
-          personal data is subject to the Privacy Policy.</p>
+        <p>Our <a href="/legal/terms-and-policies/privacy-policy">Privacy Policy</a> explain how we treat your personal data and protect your privacy when using our services. In addition, if you use the Service for more than three physical or virtual Ubuntu systems Canonical may contact you directly, using the <span class="caps">SSO </span>account details provided, to discuss ongoing support requirements.</p>
+        <p>We may also collect certain non-personally-identifiable information, which is located on your computer. The information collected may include statistics relating to how often data is transferred, and performance metrics in relation to software and configuration. You agree this information may be retained and used by Canonical.</p>
+        <p>Canonical may disclose any or all personal data and content you have sent, posted or published if required to comply with applicable law or the order or requirement of a court, administrative agency or other governmental body. All other use of your personal data is subject to the Privacy Policy.</p>
       </section>
 
       <section>
```
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: row col-8?

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html [hunk 14]
```diff
@@ -2,125 +2,72 @@
...
-          to data. Otherwise, Canonical&#39;s total liability in contract, tort or otherwise for any claims is limited to &pound;10.</p>
-
-        <p>Nothing in these Terms of Service will exclude or limit Canonical&#39;s liability for: death or personal injury caused by the negligence of Canonical; fraud or fraudulent misrepresentation; or any other liability that cannot be excluded or limited
-          by law.</p>
+        <p>Canonical will not be liable in contract, tort or otherwise for any: indirect or consequential loss; loss of profits; loss of revenue; loss of anticipated savings; loss of business or business opportunity; loss of goodwill; or loss of or corruption to data. Otherwise, Canonical&#39;s total liability in contract, tort or otherwise for any claims is limited to &pound;10.</p>
+        <p>Nothing in these Terms of Service will exclude or limit Canonical&#39;s liability for: death or personal injury caused by the negligence of Canonical; fraud or fraudulent misrepresentation; or any other liability that cannot be excluded or limited by law.</p>
       </section>
-
       <section>
```
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: row col-8?

File: templates/legal/index.html [hunk 15]
```diff
@@ -38,15 +36,14 @@ <h3 class="p-heading-icon__title">
...
-
         <div class="p-heading-icon">
           <div class="p-heading-icon__header">
             <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/03c5463a-picto-business-warmgrey.svg" alt="" />
             <h3 class="p-heading-icon__title">
-          <a href="/legal/ubuntu-advantage">
-            Ubuntu Advantage terms&nbsp;&rsaquo;
-          </a>
-        </h3>
+              <a href="/legal/ubuntu-advantage">
```
Comment (deadlight) [inline templates/legal/index.html]: The indent here seems off.

Comment (barrymcgee) [inline templates/legal/index.html]: @deadlight Nope?

screen shot 2017-06-16 at 16 09 42

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html (hunk 9 above)
Comment (barrymcgee) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: @pmahnke The whole container is wrapped in col-8? Do you think that it shouldn't be?

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html (hunk 2 above)
Comment (barrymcgee) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: Have you viewed the page locally?

File: templates/legal/bootstack/index.html (hunk 1 above)
Comment (pmahnke) [inline templates/legal/bootstack/index.html]: @barrymcgee I think this really is missing it.

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html (hunk 9 above)
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: it isn't how I have done other pages, but it renders fine... I guess you can leave it.

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html (hunk 2 above)
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: yes, both these pages look fine

File: templates/legal/terms-and-policies/privacy-policy/third-parties.html (hunk 8 above)
Comment (pmahnke) [inline templates/legal/terms-and-policies/privacy-policy/third-parties.html]: I can't find this issue now. ignor

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html (hunk 7 above)
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: ignore

File: templates/legal/terms-and-policies/livepatch-terms-of-service.html (hunk 14 above)
Comment (pmahnke) [inline templates/legal/terms-and-policies/livepatch-terms-of-service.html]: ignore

**Pull request merged**